diff --git a/src/proofofwork.py b/src/proofofwork.py
index 493b8b0..00c3b22 100644
--- a/src/proofofwork.py
+++ b/src/proofofwork.py
@@ -273,7 +273,7 @@ def init():
                 bso = None
     else:
         try:
-            bso = ctypes.CDLL(os.path.join(paths.codePath(), "bitmsghash", bitmsglib))
+            bso = ctypes.CDLL(os.path.join("/usr/lib/pybitmessage-dev-git", bitmsglib))
         except OSError:
             import glob
             try:
